crates/kornia-core/src/tensor.rs

```diff
@@ -381,6 +381,26 @@ where
         offset
     }
 
+    /// Get the index of the element at the given offset. The reverse of `Self::get_iter_offset`.
+    ///
+    /// # Arguments
+    ///
+    /// * `offset` - The offset of the element at the given index.
+    ///
+    /// # Returns
+    ///
+    /// The list of indices to get the element from.
+    pub fn get_index(&self, offset: usize) -> [usize; N] {
```

Comment:
if guess this should return an error if `offset` is larger than the number of elements

Reply:
Yes, I was wondering if this should follow the same pattern as the `get_iter_offset` method with a checked and unchecked version where the checked version returns an `Option`. What do you think?

Reply:
good idea, add test to check forth and back